mobile/geth.go

```diff
@@ -243,7 +247,12 @@ func NewNode(datadir string, config *NodeConfig) (stack *Node, _ error) {
 				config.EthereumNetworkID = int64(params.BaklavaNetworkId)
 			}
 		}
+	} else if config.EthereumNetworkID == int64(params.AlfajoresNetworkId) {
+		genesis = core.DefaultAlfajoresGenesisBlock()
+	} else if config.EthereumNetworkID == int64(params.BaklavaNetworkId) {
+		genesis = core.DefaultBaklavaGenesisBlock()
 	}
```

Comment:
Great! Just to be sure, we don't need a case for the `MainnetNetworkId` because it's already the default?

Reply:
Default was to use mainnet, but I added a case to make it more explicit.